### ISSUE TITLE
[Bug]The client is always blinking

### ISSUE THREAD

**Issue by LvShuaiChao** [lencx/ChatGPT#541]
### Version

0.11

### Bug description

https://user-images.githubusercontent.com/35528109/222628019-fc24e7bb-8ec3-4346-931e-412d6c6e8de2.mp4

### OS

windows

**Comment by Nimuthu2021** [comment]
Yeah I had the same here

**Comment by ttimasdf** [comment]
+1

**Comment by yolomoth** [comment]
same issue

**Comment by xiaop1** [comment]
same issue

**Comment by zempty-zhaoxuan** [comment]
same issue

**Comment by mqEpiphany** [comment]
same issue

**Comment by mqEpiphany** [comment]
I also have the same problem

**Comment by loveaeen** [comment]
me too

**Comment by lencx** [comment]
openai modified the page so that part of the code does not work, I will fix it in the new version.

**Comment by Nimuthu2021** [comment]
Thanks for your respond

**Comment by dimitri-vs** [comment]
In the meantime if you just want to hide the other three buttons to stop the flashing open up your console and run the code below.

Keep in mind his is a super hacky way to solve this problem, but it will do until a proper fix is implemented.

```
// Select the buttons that you want to hide
const buttons = document.querySelectorAll('#download-markdown-button, #download-png-button, #download-pdf-button');

// Create a new MutationObserver
const observer = new MutationObserver(mutations => {
  mutations.forEach(mutation => {
    if (mutation.addedNodes) {
      mutation.addedNodes.forEach(node => {
        if (node.nodeType === Node.ELEMENT_NODE && node.matches('#download-markdown-button, #download-png-button, #download-pdf-button')) {
          node.style.display = 'none';
        }
      });
    }
  });
});

// Start observing the whole document for changes
observer.observe(document, { childList: true, subtree: true });

// Hide the buttons that are already in the document
buttons.forEach(button => button.style.display = 'none');
```

**Comment by horidream** [comment]
There is a `main.js` file in the `~/.chatgpt` folder, you can put the code above to workaround this issue for now.

<img width="185" alt="image" src="https://user-images.githubusercontent.com/385359/222790866-76315891-32f7-45f1-bef8-d926631c82ab.png">

**Comment by lencx** [comment]
fix: v0.11.1

**Comment by miracleflame** [comment]
I am having the same issue and also with very similar UI used GitHub Desktop client and rarely with DeepL client. All seem to share the same framework for the UI. Pretty annoying we have longlasting bugs like these in 2025.
